sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -2607,20 +2642,19 @@ template <typename T, int NElts, lsc_data_size DS, int N>
 __ESIMD_API __ESIMD_NS::simd<T, N * NElts>
 slm_gather_impl(__ESIMD_NS::simd<uint32_t, N> offsets,
                 __ESIMD_NS::simd_mask<N> pred) {
-  detail::check_lsc_vector_size<NElts>();
-  detail::check_lsc_data_size<T, DS>();
-  constexpr uint16_t _AddressScale = 1;
-  constexpr int _ImmOffset = 0;
-  constexpr lsc_data_size _DS =
-      detail::expand_data_size(detail::finalize_data_size<T, DS>());
-  constexpr detail::lsc_vector_size _VS = detail::to_lsc_vector_size<NElts>();
-  constexpr auto _Transposed = detail::lsc_data_order::nontranspose;
-  using MsgT = typename detail::lsc_expand_type<T>::type;
+  check_lsc_vector_size<NElts>();
+  check_lsc_data_size<T, DS>();
+  constexpr uint16_t AddressScale = 1;
+  constexpr int ImmOffset = 0;
+  constexpr lsc_data_size _DS = expand_data_size(finalize_data_size<T, DS>());
+  constexpr lsc_vector_size _VS = to_lsc_vector_size<NElts>();
```

Comment:
nit: Please also remove underscore here:
```suggestion
  constexpr lsc_data_size EDS = expand_data_size(finalize_data_size<T, DS>());
  constexpr lsc_vector_size LSCVS = to_lsc_vector_size<NElts>();
```